packages/react-wrapper/package.json

```diff
@@ -11,7 +11,7 @@
     "wrapper"
   ],
   "main": "dist/index.js",
-  "types": "types/index.js",
+  "types": "types/index.d.ts",
```

Comment:
I think this types file was invalid, as inside this folder there is no index.js

Reply:
yes, good catch